bin/alias.py

```diff
@@ -40,7 +40,7 @@ def main(args):
             try:
                 print("{}={}".format(ns.expr, current_state.aliases[ns.expr]))
             except KeyError as err:
-                raise KeyError("alias: {} not found".format(err.message))
+                raise KeyError(f"alias: {err} not found")
```

Comment:
Let's not do these ruff changes in this PR.  It makes it too difficult to review.